src/features/community/merchant/actions/loadFrogs.ts

```diff
@@ -0,0 +1,127 @@
+import { communityContracts } from "features/community/lib/communityContracts";
+import frog_unrevealed from "assets/nfts/frogs/frog_unrevealed.gif";
+
+type Request = {
+  frogId: number[];
+};
+
+export type Frog = {
+  pixel_image: string;
+  attributes?: any;
+  [key: string]: any;
+};
+
+export async function loadFrogs() {
+  let frogIds = await communityContracts.getFrog().getFrogIds();
```

Comment:
Could we wrap all of this logic in a try/catch. Since this will be used in the main application, we need to support cases where the contracts/APIs could be down.

For example.

```
try {
 let frogIds = await communityContracts.getFrog().getFrogIds();
  ...
} catch {
   return null
}
```

Reply:
added a try catch on load frogs, thanks ✌🏻